src/xcode/ENA/ENA/Source/Scenes/App/AppNavigationController.swift

```diff
@@ -24,17 +24,19 @@ class AppNavigationController: UINavigationController {
 	private var scrollViewObserver: NSKeyValueObservation?
 	private var defaultScrollEdgeAppearance: UINavigationBarAppearance?
 
-	override func viewDidLoad() {
-		super.viewDidLoad()
+	override func loadView() {
+		super.loadView()
```

Comment:
Why do you do the view setup in the `loadView()` method?
I think Apple suggests doing the view config in the `viewDidLoad()` method.

Reply:
@haosap will resolve this on as duplicate of the above ...